<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExternalClockDialog</class>
 <widget class="QDialog" name="ExternalClockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>81</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>External Clock</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="ExternalClockLayout">
     <item>
      <layout class="QHBoxLayout" name="DialogLayout">
       <item>
        <widget class="QLabel" name="externalClockFrequencyLabel">
         <property name="text">
          <string>f</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ValueDialZ" name="externalClockFrequency" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>External clock frequency (Hz)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="externalClockFrequencyUnits">
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="ButtonSwitch" name="externalClockFrequencyActive">
         <property name="toolTip">
          <string>Enable or disable external clock</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/res.qrc">
           <normaloff>:/checkmark.png</normaloff>:/checkmark.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="terminateBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueDialZ</class>
   <extends>QWidget</extends>
   <header>gui/valuedialz.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>terminateBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>terminateBox</sender>
   <signal>accepted()</signal>
   <receiver>ExternalClockDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terminateBox</sender>
   <signal>rejected()</signal>
   <receiver>ExternalClockDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
